charts/app-config/templates/env-configmap.yaml

```
@@ -16,6 +16,7 @@ data:
   GOVUK_APP_DOMAIN_EXTERNAL: {{ .Values.publishingDomainSuffix }}
   GOVUK_ASSET_ROOT: https://{{ .Values.assetsDomain }}
   GOVUK_CSP_REPORT_URI: {{ .Values.cspReportURI | quote }}
+  GOVUK_ENABLE_SEARCH_AUTOCOMPLETE: "true"
```

```suggestion
  GOVUK_ENABLE_SEARCH_AUTOCOMPLETE: {{ .Values.enableSearchAutocomplete }}
```

Then add the associated value in:
https://github.com/alphagov/govuk-helm-charts/blob/main/charts/app-config/values-integration.yaml
https://github.com/alphagov/govuk-helm-charts/blob/main/charts/app-config/values-staging.yaml
https://github.com/alphagov/govuk-helm-charts/blob/main/charts/app-config/values-production.yaml

e.g.
```
enableSearchAutocomplete: "true"
```

---

Oh! That's much nicer! Thanks ❤️